example/main.js

```diff
@@ -0,0 +1,134 @@
+const namehash = require('eth-ens-namehash');
+const DnsProve  = require('./../lib/dnsprover.js');
+const DNSSEC = require("../build/contracts/DNSSEC.json");
+const ENSImplementation = require("../build/contracts/ensimplementation.json");
+const DNSRegistrar = require("../build/contracts/dnsregistrar.json");
+const Web3      = require('web3');
+const dns = require("@ensdomains/dnssec-oracle/lib/dns.js");
```

Comment:
I thought the idea was to use the DNS javascript library? dns.js was really only intended to be used for testing.